ansible/library/port_alias.py

```diff
@@ -286,17 +286,20 @@ def main():
             if portspeed_asic is not None:
                 portspeed.update(portspeed_asic)
             if front_panel_asic is not None:
-                front_panel_asic_ifnames.extend(front_panel_asic)
+                front_panel_asic_ifnames.update(front_panel_asic)
             if asicifnames_asic is not None:
                 asic = 'ASIC' + str(asic_id)
                 asic_if_names[asic] = asicifnames_asic
             if sysport_asic is not None:
                 sysports.extend(sysport_asic)
+
+        # Sort the Interface Name needed in multi-asic
+        aliases.sort(key=lambda x: int(x.split('/')[-1]))
```

Comment:
This fails if aliases have no '/'. In our linecard, the aliases are 'Ethernet0', 'Ethernet1', 'Ethernet2' etc.

Reply:
can you change your aliases name mapping ?

Reply:
@sanmalho-git Updated to to sort based on index column of port alias.